src/mbgl/gl/layer_group_gl.cpp

```diff
@@ -21,15 +21,15 @@ TileLayerGroupGL::TileLayerGroupGL(int32_t layerIndex_, std::size_t initialCapac
     : TileLayerGroup(layerIndex_, initialCapacity, std::move(name_)) {}
 
 void TileLayerGroupGL::upload(gfx::UploadPass& uploadPass) {
-    MLN_TRACE_FUNC();
-    MLN_ZONE_STR(name);
+    MLN_TRACE_FUNC()
+    MLN_ZONE_STR(name)
 
     if (!enabled) {
         return;
     }
 
-    MLN_TRACE_FUNC()
-    MLN_ZONE_STR(name)
+    // MLN_TRACE_FUNC()
+    // MLN_ZONE_STR(name)
```

Comment:
Commented because of this error:
`'___tracy_scoped_zone': redefinition; multiple initialization`